ipatests/test_webui/ui_driver.py

```diff
@@ -1163,6 +1167,24 @@ def navigate_to_row_record(self, row, pkey_column=None):
         self.wait_for_request(0.4)
         self.wait_for_request()
 
+    def navigate_to_row_record_in_new_tab(self, row, pkey_column=None):
+        """
+        Navigate to record in a new tab by CTRL + SHIFT clicking on the link
+        which brings target focused in a new tab.
+        """
+        s = 'a'
```

Comment:
Could you please change the variable name to a meaningful one?

Reply:
Thank you for suggestion, I've changed the variable name to `selector`.